localisation/00_zuwc_united_localisation.csv

```diff
@@ -3132,18 +3132,23 @@ EVTOPTA_WCPAN_200_mogu;Pandaria is my birthright! Rise mogu!;;;;;;;;;;;;;x
 EVTOPTA_WCPAN_200_generic;Pandaria stands united!;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+EVTTITLE_WCDAC_500;The Heart of the Vale;;;;;;;;;;;;;x
+EVTDESC_WCSHA_500;Some archaeologists ask for your audience. They say that opening excavation sites in the Vale of Eternal Blossoms may be a good idea. The Vale is full of titanic complexes and vaults.\n\nLegends tell that one of the vaults stores the piece of a dead Old God, Y'Shaarj. More precisely, his giant heart has been buried there after Aman'Thul heaved the Old God from the surface of the world.\n\nObtaining such an artefact will help to understand the origin and abilities of the Old Gods.;;;;;;;;;;;;;x
+EVTOPTA_WCSHA_500;Fine, I'll sponsor them.;;;;;;;;;;;;;x
+EVTOPTB_WCSHA_500;Go away.;;;;;;;;;;;;;x
+EVTDESC_WCSHA_510;Finally, you have received news from the excavation sites in the Vale of Eternal Blossoms.\n\nArchaeologists tell that they unearthed a massive building with enormous doors and security systems. It looks like the titans didn't want this vault to be ever opened. A couple of prospectors died from traps or went missing. People are fearful.\n\nWe need to spend more gold to attract workers and purchase equipment. Otherwise, the excavation has to stop.;;;;;;;;;;;;;x
+EVTOPTA_WCSHA_510;Fine, I'll invest more.;;;;;;;;;;;;;x
+EVTOPTB_WCSHA_510;Call prospectors back.;;;;;;;;;;;;;x
+EVTDESC_WCSHA_520;The lead archaeologist of the Vale of Eternal Blossoms reports to you today. They managed to open the vault.\n\nHe speaks: "When the gates were open, we felt animal fear and awe at the same time. Not many were able to enter the chamber. The heart was sealed in a giant cage chained to the ceiling and walls. We gently pulled out the heart and plunged it onto a stretcher. It wasn't an easy task to deliver the heart to the capital. Many workers were driven mad. If we plan to store it here, we must construct a new vault.";;;;;;;;;;;;;x
+EVTOPTA_WCSHA_520;The heart is mine!;;;;;;;;;;;;;x
+EVTDESC_WCSHA_550;Sadly, we have to stop excavations in the Vale of Eternal Blossoms. Your expedition no longer can work there, so we call them back.;;;;;;;;;;;;;x
```

Comment:
```suggestion
EVTDESC_WCSHA_550;Sadly, we have to stop excavations in the Vale of Eternal Blossoms. Your expedition can no longer work there, so we must call them back.;;;;;;;;;;;;;x
```

Reply:
I might be wrong here, but isn't "have had" way back in the past? I often confuse those constructions, but judging by my feeling, I would have written it without the "had", too.

Reply:
Its mostly the past, yeah, but it is also the present perfect tense, which I felt was more suitable. I used it in the sense of the excavation was stopped (in the recent past), you received notification of the stoppage and are planning to call the archaeologists back in the present.

I felt that was the best interpretation of that sentence as presumably the stoppage was related to a problem there, (be it problems with paying the workers, troubles at the site, or otherwise) making present perfect more suitable than either past or present. I might be wrong in my interpretation though.

Reply:
>  This is just me nitpicking.

Same here :) Ok, let's use your version, I can't find anything wrong on it. It's just that I very rarely use/read "have had..." or the german equivalent.

Reply:
Fixed in ce9d0e0c8aaa677cbebc34f9c3a6bf9185bb464e.